README.md

```diff
@@ -53,16 +56,17 @@ If you enjoy the app, wish to contribute or simply wish to be with like-minded p
 </p>
 
 ## 🎨 Follow [Instagram Page](https://instagram.com/quotes_status_creator)
-Follow the Instagram Page to get your daily dose of inspiration delivered to your feed! If you enjoy the app, you'll enjoy the Instagram Page too! Come follow us & be inspired!
+Follow the Instagram page to get your daily dose of inspiration delivered to your feed. \
```

Comment:
Why the carriage return at the end? Please revert if that was a mistake.

Reply:
It is intentional. That way the sentences both start on a newline.